### PR TITLE
feat: expenses_crud

### PR DESCRIPTION
**Pull request by mrdrozdy**

### CODE REVIEW

**Comment by atakowiec** [comment]
pls for God's sake, squash these commits and rebase new commits on master into your branch (before squash i guess)

**Comment by mrdrozdy** [comment]
> pls for God's sake, squash these commits and rebase new commits on master into your branch (before squash i guess)

Sorry mate, fixed innit

File: backend/src/main/java/pl/pollub/backend/categories/ExpenseCategory.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+}
```
Comment (atakowiec) [inline backend/src/main/java/pl/pollub/backend/categories/ExpenseCategory.java:29]: use lombok

File: backend/src/main/java/pl/pollub/backend/expenses/Expense.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    }
+
+    public ExpenseCategory getCategory() {
+        return category;
+    }
+
+    public void setCategory(ExpenseCategory category) {
+        this.category = category;
+    }
+}
```
Comment (atakowiec) [inline backend/src/main/java/pl/pollub/backend/expenses/Expense.java:50]: use lombok

**Review by atakowiec — CHANGES_REQUESTED** [review]
use lombok pls

**Comment by atakowiec** [comment]
c'mon daddy, give me this fix

**Comment by mrdrozdy** [comment]
> c'mon daddy, give me this fix

Done it mate innit

**Review by atakowiec — APPROVED** [review]
approved, merge to main

**Pull request merged**